css/selectors/matches.json

```
@@ -8,60 +8,103 @@
           "support": {
             "chrome": [
               {
-                "version_added": "66"
+                "version_added": "66",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "Experimental Web Platform Features",
+                    "value_to_set": "enabled"
+                  }
+                ],
+                "notes": "Has issues with combinators (see <a href='https://crbug.com/842157'>bug 842157</a>)."
               },
               {
                 "version_added": "12",
-                "alternative_name": ":-webkit-any"
+                "alternative_name": ":-webkit-any",
+                "notes": "Doesn't support combinators."
               }
             ],
             "chrome_android": [
               {
-                "version_added": "66"
+                "version_added": "66",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "Experimental Web Platform Features",
+                    "value_to_set": "enabled"
+                  }
+                ],
+                "notes": "Has issues with combinators (see <a href='https://crbug.com/842157'>bug 842157</a>)."
               },
               {
                 "version_added": "18",
-                "alternative_name": ":-webkit-any"
-              }
-            ],
-            "edge": [
-              {
-                "version_added": "15"
-              },
-              {
-                "version_added": "12",
-                "alternative_name": ":-ms-matches"
+                "alternative_name": ":-webkit-any",
+                "notes": "Doesn't support combinators."
               }
             ],
+            "edge": {
+              "version_added": false
+            },
             "edge_mobile": {
               "version_added": false
             },
             "firefox": {
               "version_added": "4",
-              "alternative_name": ":-moz-any"
+              "alternative_name": ":-moz-any",
+              "notes": "Doesn't support combinators."
```

Nit: Consider adding a link to [bug&nbsp;906353](https://bugzilla.mozilla.org/show_bug.cgi?id=906353):
```suggestion
              "notes": [
                "Doesn't support combinators.",
                "See <a href='https://bugzil.la/906353>bug 906353</a>"
              ]
```

---

Added. (GitHub didn't offer me to accept your suggestion directly, as I'm not the PR author. [I think they should allow maintainers to accept these, too])